include/tester/db/VTDatabase.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "Database.hpp"
+
+class VTDatabase : Database
```

Comment:
Remember that there are also inheritance modifiers - understand why it is important to mark this `public` here.

Reply:
Done